packages/client/hmi-client/src/components/widgets/tera-related-publications.vue

```diff
@@ -119,8 +120,7 @@ const addResources = () => {
 
 const sendForEnrichments = async (/* _selectedResources */) => {
 	// 1. Send dataset profile request
-	/* TODO: send selected resources along with dataset to backend for enrichment */
-	const resp = await profileDataset(props.assetId);
+	const resp = await profileDataset(props.assetId, selectedResources.value.id);
```

Comment:
Be careful that `selectedResources` can be null, a user is not obliged to select one.

Reply:
i've surrounded it in a if to check if it is null now

Reply:
scratch that ^.  Now defaults to null if there is no selection